data/json/items/ranged.json

```diff
@@ -2938,7 +2938,7 @@
     "name": "M72 LAW",
     "name_plural": "M72 LAW",
     "description": "A disposable light anti-tank rocket launcher.  Once fired, it cannot be reloaded and must be disposed of.",
-    "price": 850000,
+    "price": 90000, "//": "A tube with a lever. Unsurprisingly, dirt cheap (at least for a rocket launcher).",
```

Comment:
Full-auto weapons were generally tougher to get, and if you'll look at the M2010 or thereabouts, there's a huge markup on it when sold on the civilian market.  LAW rocket has the same issue.  I never intended it or the RPG to be available on the civilian (open) market.

(If nothing else, it's the joke about bullet control, rather than gun control.  LAWs, though obsolete, are unavailable so folks don't use 'em to take out the local PD.  Civil Defense is one thing, insurrection is another.)